### PR TITLE
Dor 280 private beta start page

### PR DESCRIPTION
**Pull request by slashrealm**

### CODE REVIEW

File: app/uk/gov/hmrc/traderservices/views/PrivateBetaStartPageView.scala.html [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+@govukLayout(
+  pageTitle = pageTitle("view.beta.title")) {
+
+  @html.h1(messages("view.beta.heading"), classes = "govuk-heading-l")
+  @html.p { @messages("view.beta.lede") }
+
+  <aside>
+    <nav class="contents-list">
+      <h2 class="contents-list__title" id="contents">@messages("view.beta.contents")</h2>
```
Comment (Tasmine10ten) [inline app/uk/gov/hmrc/traderservices/views/PrivateBetaStartPageView.scala.html:34]: Can see "Contents" is a H2 but shows like this on the page 
image

File: conf/messages.en [hunk 2]
```diff
@@ -608,6 +608,44 @@ view.amend-case.summary.documents.heading=Documents you are submitting
...
+view.beta.howToUpload.fileTypes2=PDF (.pdf)
+view.beta.howToUpload.fileTypes3=email (.txt, .oft or .msg)
+view.beta.howToUpload.paragraph3=You cannot upload Microsoft files, zip files or documents that are password-protected.
+view.beta.howToUpload.paragraph4=If you have a Word or Excel document to upload, you need to convert it to PDF first.
+view.beta.howToUpload.paragraph5=If you’ve already sent your documents for clearance and need to add additional information, you must have the case reference number. You will have received this number when the case was first created.
+view.beta.beforeYouStart=Before you start
+view.beta.beforeYouStart.paragraph1=You will need a user ID and password. If you do not have a user ID, you can create one the first time you send us your documents for customs clearance.
+view.beta.beforeYouStart.startNow=Start now
+view.beta.afterSendingDocuments=After you’ve sent HMRC supporting documents
+view.beta.afterSendingDocuments.paragraph1=After you have completed the service you'll be told when your goods should be cleared. You’ll be notified through your software when they have cleared.
```
Comment (Tasmine10ten) [inline conf/messages.en:646]: First apostrophe needs to be a smart one

File: app/uk/gov/hmrc/traderservices/views/PrivateBetaStartPageView.scala.html (hunk 1 above)
Comment (slashrealm) [reply app/uk/gov/hmrc/traderservices/views/PrivateBetaStartPageView.scala.html:34]: Yes, this is the correct GDS pattern, eg. https://www.gov.uk/eori.
It was discussed with Meg and she was happy with it.

File: conf/messages.en (hunk 2 above)
Comment (slashrealm) [reply conf/messages.en:646]: :+1:

**Pull request merged**